lib/livebook/hubs/personal.ex

```diff
@@ -7,6 +7,7 @@ defmodule Livebook.Hubs.Personal do
   alias Livebook.Hubs
 
   @secret_key_size 64
+  @prefix "lb_ph_"
```

Comment:
We also need a migration that adds the prefix to existing keys.

Reply:
I don't think we need to prefix the personal hub. We will never be able to report it back on stuff like Github Security Scanning, so I would just remove it?

Reply:
@josevalim ah, checking the format itself is too brittle?

Reply:
The goal of checking is to be able to report it and remove it. But because we do know from where it came from, we can't do anything. :D

Reply:
I see, good call!